packages/core/src/lib/Cache.ts

```diff
@@ -97,10 +117,10 @@ export class MemoryCache implements ICache {
     return this.readWithExactlyOneFetch(key, () => read(current.value()))
```

Comment:
ttl missing?

Reply:
Currently we don't have any ttl semantics on the cache, only LRU. Default size is 1M items which depending on the size of the state can be anywhere from a few megabytes when full up to a few gigabytes of memory usage. (a million kilobytes is a gigagbyte). So I think the question is whether to even have TTL support, or whether to ask people to configure sane size caps on their cache. JS doesn't have a way to inspect the byte size of an object either so there's no good way to determine the memory usage of the cache. There's no inherent harm in more memory usage either, if you're running on k8s you'll have configured a memory request and limit for your process, if you have at any point allocated `N` MB then that'll be reserved for your process going forward, so just because we free the memory doesn't actually mean another process can start using it. Lots of stuff to think about with this!

Reply:
🤔